cmd/cluster-operator-controller-manager/app/controllermanager.go

```diff
@@ -338,7 +339,9 @@ func NewControllerInitializers() map[string]InitFunc {
 	controllers["nodeconfig"] = startNodeConfigController
 	controllers["deployclusterapi"] = startDeployClusterAPIController
 
+	// Controllers ported to upstream cluster API
```

Comment:
What does it mean for the controllers to be "ported to upstream cluster API"? At best this comment is confusing. At worst it is inaccurate.

Reply:
PR started out as a port of cluster controller to use the new API, and has been through many iterations since. I will update it.

Please keep tone of PR comments in mind.